tools/cli/workflow_commands.go

```diff
@@ -526,12 +526,15 @@ func processHeader(c *cli.Context) (map[string][]byte, error) {
 	headers := mapFromKeysValues(headerKeys, headerValues)
 	// append context headers if exist
 	if span := opentracing.SpanFromContext(c.Context); span != nil && span.Context() != nil {
-		span.Context().ForeachBaggageItem(func(k, v string) bool {
```

Comment:
how did you find out this doesn't work? Does the unit test change cover it?

Reply:
When I test with internal tracer, I realize the headers key doesn't match. Now the unit test coverage covered it. (prefix `mockpfx-baggage-`)